app/controllers/users_controller.rb

```diff
@@ -35,24 +35,34 @@ def update
   end
 
   def destroy
+    User.find(params[:id]).destroy
```

Comment:
Change to `User.destroy(params[:id])`.
http://apidock.com/rails/v2.3.8/ActiveRecord/Base/destroy/class